web/webpack.config.js

```diff
@@ -19,6 +19,7 @@ const config = {
   },
   output: {
     path: path.join(__dirname, 'dist'),
+    publicPath: '/',
```

Comment:
Adding this causes webpack to prepend `/` to the paths it injects into our HTML. This is important with the paginated view because without it, hitting reload on a page would cause an error. The page would load, but the CSS and JS would not, so you'd just get a white page with no explanation of what was going on. I think this is the fix, but we'll see...